petitioner_app/templates/uploader/settings.yaml

```diff
@@ -0,0 +1,14 @@
+client_config_backend: file
+client_config:
+    client_id: <your_client_id>
```

Comment:
Tutaj nam przekażesz jakieś credentialsy czy każdy na swoim drivie będzie odpalał? Pytanie jak łatwiej

Reply:
Myślę że od razu można wstawić jakiś. Pewnie ja będę prezentował ale jak już Mateusz się zajmuje google drive to dobrze by było żeby to uzupełnił od razu żeby nie mnożyć roboty.